client/src/Components/Register/index.jsx

```diff
@@ -0,0 +1,52 @@
+import React, {Component} from 'react';
+
+class Register extends Component {
+    render() {
+        return (
+            <header>
+                <div className="container-fluid">
+                    <div className="bg-img">
+                    </div>
+                    <div className="row-fluid">
+                        <div className="LogoInSignUp">
+                            <h1>Sport News</h1>
+                        </div>
+                        <div className="LogIn">
+                            <h3>Already have an account?</h3>
+                            <a href="#" className="btn btn-primary login">Log In</a>
+                        </div>
+                    </div>
+                    <div className="ca-form">
+                        <div className="ca-elements">
+                            <h2>Create Account</h2>
+                            <a href="#" className="btn fb-icon"></a>
+                            <a href="#" className="btn gplus-icon"></a>
+                            <p>Or use your email for registration</p>
+                            <div className="first-name">
+                                <h5>first name</h5>
+                                <br/>
```

Comment:
it is bad practice to use <br>. It is better to add margin for example

Reply:
reworked